packages/ide-extension/src/panel/guidedAnswersPanel.ts

```diff
@@ -161,9 +161,8 @@ export class GuidedAnswersPanel {
         }
         if (this.startOptions) {
             await this.processStartOptions(this.startOptions);
-        } else {
-            await this.processEnvironmentFilters(this.ide);
         }
+        await this.loadQuickFilters(this.ide);
```

Comment:
If Guided Answers extension is started with arguments (`this.startOptions`) to load a certain tree and node path, I think we should show what was passed as argument and NOT overwrite it with quickfilters. That is why it previously was in the `else`-branch.

Reply:
Great find, and also a good reminder that we need to extend the node versions for pipeline builds. Currently only 
```
node-version: [16.x, 18.x]
```
https://github.com/SAP/guided-answers-extension/blob/5af480cd96bf64e31bc3cc139906630ecc515bb4/.github/workflows/pipeline.yml#L14C20-L14C20

I'll create a separate PR for this.

Reply:
Sorry, I missed the part that behavior of quick filters slightly changed, they are not loaded on start anymore. Saw that now in the ticket.